backends/jgroups/src/test/java/org/hibernate/search/backend/jgroups/impl/SyncJGroupsBackendTest.java

```diff
@@ -74,52 +76,20 @@ public void testSynchAsConfigured() {
 		JGroupsReceivingMockBackendQueueProcessor dvdBackendMock = extractMockBackend( "dvds" );
 
 		dvdBackendMock.resetThreadTrap();
-		boolean timeoutTriggered = false;
 		try {
 			//DVDs are sync operations so they will timeout:
 			final long presendTimestamp = System.nanoTime();
-			log.trace( "[PRESEND] Timestamp: " + presendTimestamp );
+			System.out.println( "[PRESEND] Timestamp: " + presendTimestamp );
 			storeDvd( 1, "Hibernate Search in Action" );
 			final long postsendTimestamp = System.nanoTime();
 			final long differenceInMilliseconds = TimeUnit.MILLISECONDS.convert( (postsendTimestamp - presendTimestamp), TimeUnit.NANOSECONDS );
-			log.trace( "[POSTSEND] Timestamp: " + postsendTimestamp + " Diff: " + differenceInMilliseconds + " ms." );
-		}
-		catch (SearchException se) {
-			//Expected: we're inducing the RPC into timeout by blocking receiver processing
-			Throwable cause = se.getCause();
-			Assert.assertTrue( "Cause was not a TimeoutException but a " + cause, cause instanceof TimeoutException );
-			timeoutTriggered = true;
+			System.out.println( "[POSTSEND] Timestamp: " + postsendTimestamp + " Diff: " + differenceInMilliseconds + " ms." );
```

Comment:
Yes you understand correctly. This is the very problem that led me to take additional days to investigate :(
Apparently now when doing a multi-cast, if some (any) nodes don't answer it just moves on. One should build the backend differently. It will still attempt to deliver the messages in the right order automatically, but nodes in the cluster group are rather allowed to "lag behind" than to block it all.

Therefore the previous assertions no longer make sense. I was tempted to either:
 - remove the test
 - rewrite the whole backend differently

Yet while working further on it, this test actually prevented me from making some silly mistakes - so while I agree it's all puzzling I'd rather keep it now, move on, and see if we can make some time to rewrite the whole thing.